client/src/components/brushableHistogram/index.js

```diff
@@ -523,6 +523,7 @@ class HistogramBrush extends React.PureComponent {
           </span>
           <span
             data-testclass="brushable-histogram-field-name"
+            data-testid={`${field}:field-name`}
```

Comment:
`:field-name` is seems overly generic - we may have other fields in the future.  Maybe something more descriptive?

Reply:
Ah! Turns out I didn't use this anyway. Deleting it.